src/components/NurseryWithdrawals/NurseryPlantingsAndWithdrawals.tsx

```diff
@@ -55,37 +58,32 @@ export default function NurseryPlantingsAndWithdrawals({ reloadTracking }: Nurse
   }, [tab, activeLocale]);
 
   return (
-    <TfMain>
-      <Box sx={{ paddingLeft: theme.spacing(3) }}>
-        <Grid container spacing={3} sx={{ marginTop: 0 }}>
-          <PageHeaderWrapper nextElement={contentRef.current}>
-            <Grid container spacing={3} sx={{ paddingLeft: theme.spacing(3), paddingBottom: theme.spacing(4) }}>
-              <Grid item xs={8}>
-                <Typography sx={{ marginTop: 0, marginBottom: 0, fontSize: '24px', fontWeight: 600 }}>
-                  {strings.WITHDRAWALS}
-                </Typography>
-              </Grid>
-            </Grid>
-          </PageHeaderWrapper>
-          <Grid item xs={12}>
-            <PageSnackbar />
-          </Grid>
-          <Box ref={contentRef} display='flex' flexDirection='column' flexGrow={1}>
-            <Tabs
-              activeTab={activeTab}
-              onTabChange={onTabChange}
-              tabs={[
-                {
-                  id: 'planting_progress',
-                  label: strings.PLANTING_PROGRESS,
-                  children: <PlantingProgress reloadTracking={reloadTracking} />,
-                },
-                { id: 'withdrawal_history', label: strings.WITHDRAWAL_HISTORY, children: <NurseryWithdrawals /> },
-              ]}
-            />
-          </Box>
+    <Box sx={{ paddingLeft: theme.spacing(3) }}>
+      <Grid container spacing={3} sx={{ marginTop: 0 }}>
+        <Grid item xs={12}>
+          <PageSnackbar />
```

Comment:
i made a bunch of changes here, can you rebase.. we want the map to have full height .. or if this has diverged too much if you can maintain full height it would be fine

Reply:
right now the map has lost full width as well.. fix both? width and height

Reply:
I will try to rebase